pkg/service/serve/graph/schema.resolvers.go

```diff
@@ -8,6 +8,9 @@ import (
 	"context"
 	"fmt"
 
+	sentry "github.com/getsentry/sentry-go"
```

Comment:
Can you explain the changes in file ..?

Reply:
So in this file, we are just starting the tests and returning from the function, to check if the test run has completed or not, a testetstatus function has been written to continuosly check the status of the test run, so I've added my telemetry events there to trigger when the status changes to "PASSED" or "FAILED".  Apart from that, I've added Sentry to the goroutines.